tests/api/worlds/distance_sensor_infra-red_cached_images.wbt

```diff
@@ -0,0 +1,57 @@
+#VRML_SIM R2020b utf8
```

Comment:
```suggestion
#VRML_SIM R2021a utf8
```

Reply:
Unfortunately, I cannot apply this directly on this branch... Creating a new branch...

Reply:
Oh yes right :sob: